server/setup_server.sh

```diff
@@ -56,7 +55,7 @@ else
     # restart server weekly to ensure it uses the latest certificates (certbot renews them automatically)
     echo "
     PATH=/bin:/usr/bin:/usr/local/bin
-    @weekly env HOME=$HOME pm2 restart BraveServer
+    @weekly env pm2 restart BraveServer
```

Comment:
since the `HOME=$HOME` has been removed, I don't think the `env` call does anything, but I'm not totally sure about this.

Reply:
Cool, I'll try that out next week.

I think I should also add a test Button press to my definition of a "working state".